adapters/celestia/src/verifier/address.rs

```diff
@@ -91,4 +142,90 @@ mod tests {
         let output = format!("{}", address);
         assert_eq!(raw_address_str, output);
     }
+
```

Comment:
Let's add a test with a real celestia address and make sure it encodes/decodes correctly. I stole this one from the block explorer: `celestia1hvp2nfz3r6nqt8mlrzqf9ctwle942tkr0wql75`

Reply:
Discussed over zoom